src/variables.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --default-padding: 16px;
+  --default-border: 1px;
+  --default-width: calc(200px - 2 * var(--default-padding)
```

Comment:
Too complex variable. The dimensions may change and the layout may be damaged during changes